libs/features/personalization/personalization.js

```diff
@@ -786,6 +786,7 @@ export function cleanAndSortManifestList(manifests) {
         freshManifest.name = fullManifest.name;
         freshManifest.selectedVariantName = fullManifest.selectedVariantName;
         freshManifest.selectedVariant = freshManifest.variants[freshManifest.selectedVariantName];
+        delete freshManifest.variants;
```

Comment:
looks like this one is the only business logic change. Everything else is a name swap. Is this prop dele intentional ?

Reply:
yes, this is intentional. I just updated the comment to mention it. Thanks.

Reply:
@markpadbe I hate to say it, but let's not remove.  I forgot about this use of the object.  We're essentially trying to take the decision from the Target instance and merge it with the fresh data of the imported instance (when there's a duplicate).  So I worry if we delete the variants we create a small fringe case of deleting it and then needing it later in the loop.